src/functional_test_runner/lib/proc_runner/proc.js

```diff
@@ -0,0 +1,65 @@
+import { spawn } from 'child_process';
+import { statSync } from 'fs';
+
+import Rx from 'rxjs/Rx';
+import { brightBlack } from 'ansicolors';
+import treeKill from 'tree-kill';
+import { fromNode as fcb } from 'bluebird';
```

Comment:
This should be able to used the built-in `promisify` in Node 8 instead of `fromNode` in `bluebird` (finally! :tada:), maybe something along the lines of this should work:

```
import treeKill from 'tree-kill';
import { promisify } from 'util';

const treeKillAsync = promisify(treeKill);

await treeKillAsync(...);
```